webapp/portlet/src/main/webapp/vue-apps/attach-image/components/form/ImageInput.vue

```diff
@@ -71,6 +72,11 @@ export default {
   },
   created() {
     document.addEventListener('attachment-save', this.triggerAttachmentsSave);
+    this.$root.$on('attachments-update-cropped-image', (event) => {
```

Comment:
Please, avoid using generic listeners on input events. What is the same input is added twice in the same form? Which one should intercept this event ?

```suggestion
```

Reply:
cc @boubaker